guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -0,0 +1,33 @@
+[id="requirements-for-installation-in-an-ipv6-network_{context}"]
+= Requirements for {Project} Installation in an IPv6 Network 
+
+Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
+
+ifeval::["{build}" == "satellite"]
+* {SmartProxy} must be installed on {RHEL} version 7.9 or higher.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+* {SmartProxy} must be installed on a system with grub2 version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher. For other operating systems, copy newest grub build to `/var/lib/tftpboot/grub2/grubx64.efi`.
+endif::[]
+
+* {Project} or {SmartProxy} can be only installed on IPv6-only systems, dual-stack is known to work but not supported.
```

Comment:
I'd keep this downstream only. Upstream we don't really have this distinction. If it's know to work, users can use it.

Reply:
Done! Thank you.